tests/e2e/kueue_test.go

```diff
@@ -20,10 +28,31 @@ func kueueTestSuite(t *testing.T) {
 
 	t.Run("Validate component enabled", componentCtx.ValidateComponentEnabled)
 	t.Run("Validate operands have OwnerReferences", componentCtx.ValidateOperandsOwnerReferences)
+	t.Run("Validate Kueue Dynamically create VAP", componentCtx.validateKueueVAPReady)
 	t.Run("Validate update operand resources", componentCtx.ValidateUpdateDeploymentsResources)
 	t.Run("Validate component disabled", componentCtx.ValidateComponentDisabled)
 }
 
 type KueueTestCtx struct {
 	*ComponentTestCtx
 }
+
+func (tc *KueueTestCtx) validateKueueVAPReady(t *testing.T) {
+	g := tc.NewWithT(t)
+	v, err := tc.GetClusterVersion()
+	g.Expect(err).ToNot(HaveOccurred())
+	if v.GTE(semver.MustParse("4.17.0")) {
+		g.Get(gvk.ValidatingAdmissionPolicy, types.NamespacedName{Name: "kueue-validating-admission-policy"}).Eventually().WithTimeout(1 * time.Second).
```

Comment:
Is there any specific reason to wait only 1 second ?

Reply:
i moved this test case after "ValidateUpdateDeploymentsResources" test.
i think no need to wait DefaultTimeout 2mins here, if component is ready but VAP/VAPB not found